sdk/appconfiguration/app-configuration/src/internal/synctokenpolicy.ts

```diff
@@ -29,7 +30,7 @@ export function syncTokenPolicy(syncTokens: SyncTokens): PipelinePolicy {
       if (syncTokenHeaderValue) {
         request.headers.set(SyncTokenHeaderName, syncTokenHeaderValue);
       }
-
+      logger.info("[syncTokenPolicy] Getting the next request");
```

Comment:
misleading.. suggest removing this.

This is more of.. passing on the request to the next policy.

Reply:
Instead add a log inside the if check saying the header is added.